src/styles/_personKort.less

```diff
@@ -1,5 +1,5 @@
 .personkort {
-  margin-bottom: 1em;
+  margin-bottom: 0.5rem;
```

Comment:
Hvorfor `rem` i stedet for `em`?

Reply:
Blir ikke helt klok på rem og em selv, men jeg har sett at tailwind bruker rem når vi skriver `mb-4` = `1rem`, så da tenkte jeg det gir mest mening. Vet vi har brukt em før 🤷🏼‍♂️
Virker som at rem er mer riktig på globalt nivå, mens em er mer når man vil at noe skal skalere i forhold til sin parent?
https://refine.dev/blog/rem-vs-em/#when-to-use-em-and-rem-units-in-css

Reply:
Også går det an å skrive bare `.5` 👍